brontology/graph/entity.py

```diff
@@ -1,11 +1,28 @@
 # noinspection PyUnresolvedReferences
-from typing import Union, Optional
-
+from typing import Union, Optional, List, Tuple
+from random import choice
+from math import sqrt
 from brontology.graph.node import Node, Link
 
+Vector = Tuple[float]
 
 class Synset:
-    ...
+    def __init__(self, embeddings: List[Vector]):
+        self.embeddings = embeddings
+        self.center = self.calculate_center()
+        self.repr = self.get_representative_word()
+
+    def calculate_center(self) -> Vector:
+        # Berechnung des Zentrums der Vektoren
```

Comment:
Die DocStrings für alle 3 Methoden müssen noch zu "echten" DocStrings umgewandelt werden (Und in englisch geschrieben werden!)

Dazu schreibt man den Text einfach in 3 Anführungszeichen statt dem Hashtag-Kommentar.
Der Unterschied zwischen Line- und Block-Kommentaren ist in Python etwas verwirrend, aber das ist er, und für DocStrings nimmt man immer Blockkommentaren.

In diesem Fall wäre das dann einfach neu:
`"""Berechnung des Zentrums der Vektoren"""` (Final in englisch natürlich)